src/Features/CSharp/Portable/LanguageServices/CSharpStructuralTypeDisplayService.cs

```diff
@@ -18,17 +18,17 @@
 
 namespace Microsoft.CodeAnalysis.Editor.CSharp.LanguageServices
 {
-    [ExportLanguageService(typeof(IAnonymousTypeDisplayService), LanguageNames.CSharp), Shared]
-    internal class CSharpAnonymousTypeDisplayService : AbstractAnonymousTypeDisplayService
+    [ExportLanguageService(typeof(IStructuralTypeDisplayService), LanguageNames.CSharp), Shared]
+    internal class CSharpStructuralTypeDisplayService : AbstractStructuralTypeDisplayService
```

Comment:
most of the code changes are a strictly mechanical change of 'AnonymousType' => 'StructuralType'.  I'll mark which parts of the code are actually changed.